modules/widgets/facebook-likebox.php

```diff
@@ -104,7 +104,7 @@ function update( $new_instance, $old_instance ) {
 			'href'        => trim( strip_tags( stripslashes( $new_instance['href'] ) ) ),
 			'height'      => (int) $new_instance['height'],
 			'show_faces'  => (bool) $new_instance['show_faces'],
-			'stream'      => (bool) $new_instance['stream'],
+			'stream'      => (bool) $new_instance['stream'] = false,
```

Comment:
This will overwrite the option always. We should be checking if there is a new instance of it before we write it.

something like `isset( $new_instance['stream'] ) ? (bool) $new_instance['stream'] : false,`